Dockerfile

```diff
@@ -21,13 +28,22 @@ ENV DEBUG="False" \
     ENABLE_CAPTCHA="False" \
     LEGAL_LINK="False"
 
-RUN mkdir -p /etc/ihatemoney &&\
-    pip install --no-cache-dir gunicorn pymysql;
-
 ADD . /src
 
-RUN pip install --no-cache-dir -e /src
+RUN echo "**** install runtime packages ****" && \
+    apk add --no-cache \
+    shadow && \
+    echo "**** create runtime folder ****" && \
+    mkdir -p /etc/ihatemoney &&\
+    echo "**** install pip packages ****" && \
+    pip install --no-cache-dir gunicorn pymysql && \
```

Comment:
How about installing `psycopg2` too, so you can use a postgres database with the docker image?

Reply:
I tried adding it afterwards by running from a simple local Dockerfile:
```Dockerfile
FROM ihatemoney/ihatemoney:5.1.1
RUN pip install --no-cache-dir psycopg2
```

but building fails, not sure why, I don't really have experience using python.

Reply:
Can you tell us how it fails, maybe you have some logs?

Reply:
Sure, I've opened a proper Issue: #924